pyomo/opt/testing/pyunit.py

```diff
@@ -32,13 +32,16 @@ def _failIfPyomoResultsDiffer(self, cmd=None, baseline=None, cwd=None):
             baseline = "\n".join(INPUT.readlines())
             INPUT.close()
     
-        output = pyutilib.subprocess.run(cmd)
+        output = subprocess.run(cmd, stdout=subprocess.PIPE,
+                                stderr=subprocess.STDOUT)
```

Comment:
Should we switch this over to using `universal_newlines=True` here and avoid the call to `decode()` on line 44?